CHANGES.rst

```diff
@@ -69,6 +69,12 @@ ramp_fitting
   of groups in a segment are computed when applying optimal weighting to line
   fit segments. [#7560, spacetelescope/stcal#163]
 
+srctype
+-------
+
+- The SRCTYPE takes precedence over PATTTYPE when setting the source type for
```

Comment:
I believe you mean SRCTYAPT here, as opposed to SRCTYPE? The former is the input to the step, the latter is the output.